packages/api/src/router/developer.ts

```diff
@@ -5,3 +5,39 @@ export const developerRouter = createTRPCRouter({
     return await ctx.db.developers.count();
   }),
 });
+
+export const ryanLiRouter = createTRPCRouter({
+  info: publicProcedure.query(async ({ ctx }) => {
```

Comment:
when you merge into developer router, rename the procedures to ryanli_info instead